core/iwasm/common/wasm_exec_env.c

```diff
@@ -74,6 +74,12 @@ wasm_exec_env_create_internal(struct WASMModuleInstanceCommon *module_inst,
 #if WASM_ENABLE_MEMORY_TRACING != 0
     wasm_runtime_dump_exec_env_mem_consumption(exec_env);
 #endif
+
+#if WASM_ENABLE_FAST_JIT != 0
+    /* always double check */
+    exec_env->jit_cache_len = 4 * sizeof(uint64);
+    memset(exec_env->jit_cache, 0, exec_env->jit_cache_len);
```

Comment:
No need to add jit_cache_len, use `memset(exec_env->jit_cache, 0, sizeof(exec_env->jit_cache))`

Reply:
consider adding a runtime cache length checking in the future to prevent overwriting the content in the `jit_cache`